internal/devserver/ts_devserver.bzl

```diff
@@ -84,6 +102,11 @@ ts_devserver = rule(
         "serving_path": attr.string(),
         "data": attr.label_list(allow_files = True, cfg = "data"),
         "static_files": attr.label_list(allow_files = True),
+        "concat_files": attr.label_list(allow_files = True),
+        # @// is special syntax for the "main" repository
+        # The default assumes the user specified a target "node_modules" in their
+        # root BUILD file.
+        "node_modules": attr.label_list(default = [Label("@build_bazel_rules_typescript//:node_modules"), Label("@//:node_modules")]),
```

Comment:
it will be hard for user to override this default, because they'll need to know about their own node_modules as well as build_bazel_rules_typescript. nodejs rules have a single-valued node_modules.
So I think it would be better to keep these as two separate attributes
_rules_typescript_node_modules = attr.label(default=....
node_modules = attr.label(default=...

Reply:
Ok. I agree.